test/test_playbooks/tasks/content_view.yml

```diff
@@ -11,6 +11,7 @@
     password: "{{ foreman_password }}"
     server_url: "{{ foreman_server_url }}"
     verify_ssl: "{{ foreman_verify_ssl }}"
+    state: "{{ content_view_state }}"
```

Comment:
I think, ` | omit` cannot be wrong here. The state has a default value.

I know people, who use these tasks elsewhere. ;) @akofink